### ISSUE TITLE
elevated fetal middle cerebral artery dopplers

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7007]
**Preferred term label:**
elevated fetal middle cerebral artery dopplers

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**
fetal middle cerebral artery dopplers that are elevated during pregnancy

**Parent term (use hpo.jax.org/app)**
N/A

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
fetal anemia

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by pnrobinson** [comment]
@sparkste  What about this
label: Elevated fetal middle cerebral artery doppler peak systolic velocity  
synonym Elevated fetal MCA-PSV  
def: Increased peak systolic velocity of the fetal middle cerebral artery (MCA) as evaluated by Doppler ultrasound.
comment: The technique for middle cerebral artery peak systolic velocity measurement involves an axial section through the fetal head that includes the thalami and cavum septum pellucidum is obtained and the circle of Willis visualized with color or power Doppler. The middle cerebral artery is examined close to its origin from the internal carotid artery. The sample volume should be placed in the center of the vessel and the angle between the ultrasound beam and the direction of blood flow kept as close to 00 as possible. Angle correction can be used if required. The highest peak systolic waveform is measured, in the absence of fetal breathing movements. PMID:28191060

Would this https://hpo.jax.org/app/browse/term/HP:0010948
be a good parent term?

**Comment by sparkste** [comment]
A couple of minor edits that I would suggest:
"comment: The technique for MCA peak systolic velocity measurement involves an axial section through the fetal head that includes the thalami and cavum septum pellucidum is obtained and the circle of Willis visualized with color or power Doppler. The MCA is examined close to its origin from the internal carotid artery. The sample volume should be placed in the center of the vessel and the angle between the ultrasound beam and the direction of blood flow kept as close to 0 degrees as possible. Angle correction can be used if required. The highest peak systolic waveform is measured, in the absence of fetal breathing movements. An elevated MCA Doppler measurement indicates that the fetus is likely to be anemic, but this measurement is not perfectly sensitive or specific. PMID:28191060"

I actually don't think that abnormality of the fetal cardiovascular system is a clear parent term. To me, this sounds like a structural abnormality of the fetal cardiovascular system (unless it is not intended this way). Elevated MCA Dopplers in a fetus indicate fetal anemia, rather than a structural difference. Is there an alternative parent term?

Thanks so much!
Teresa



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by patt1morgan** [comment]
@TonyRoscioli, @DominicIliescu - Greetings!  As you will see, there has already been some discussion around this new term request.  It would be helpful if you could give us any suggestions you may have before we finalize this addition to the HPO. Thank you in advance for taking the time to review...

**Comment by TonyRoscioli** [comment]
Can we discuss what to do please ?

Is there a website for comment?

Sent from my iPhone

On 20 Jul 2021, at 12:30 am, patt1morgan ***@***.***> wrote:

﻿

@TonyRoscioli<https://github.com/TonyRoscioli>, @DominicIliescu<https://github.com/DominicIliescu> - Greetings! As you will see, there has already been some discussion around this new term request. It would be helpful if you could give us any suggestions you may have before we finalize this addition to the HPO. Thank you in advance for taking the time to review...

**Comment by patt1morgan** [comment]
@TonyRoscioli -  Hi Tony,  You are welcome to comment in detail in the 'comment' box below.  If further discussion is required.  Peter will be reviewing, in depth, all of the comments before finalizing definitions placed in the HPO.  If you have further questions or would like to discuss directly with Peter, he can be 'tagged' @pnrobinson (also in the comment box below). Thank you again for your involvement in this project. We look forward to 'picking your brain' as we proceed.

**Comment by DominicIliescu** [comment]
Hello everyone!
Very good description above of the sampling method, nothing to add.
Regarding the term label, the most comprehensive was posted above by pnrobinson: Elevated fetal middle cerebral artery doppler peak systolic velocity. However, it may be reduced, as the velocities are measured prenatally only by Doppler means. Therefore, a variant of the term label may read: elevated fetal middle cerebral artery peak systolic velocity

**Comment by pnrobinson** [comment]
@sparkste Added this term with a new parent term Abnormal fetal cardiovascular physiology
